lib/velum/salt.rb

```diff
@@ -12,8 +12,8 @@ class Salt
     # Returns two values:
     #   - The response object.
     #   - A hash containing the parsed JSON response.
-    def self.call(action:, targets: "*", arg: nil)
-      hsh = { tgt: targets, fun: action, client: "local" }
+    def self.call(action:, targets: "*", target_type: "glob", arg: nil)
```

Comment:
No, because by default `target_type` was already `glob`, and we were using it this way. We are extending the functionality to allow other `expr_type` here, but still using the default which is `glob`.